tests/test_backend.py

```diff
@@ -1248,6 +1760,35 @@ def test_items(self):
             self.assertEqual(item['tag'], 'test')
             self.assertEqual(item['classified_fields_filtered'], None)
 
+    @unittest.mock.patch('perceval.backend.sys.stderr')
```

Comment:
These kind of tests can be removed. What I miss is a test for `TestBackend` to check that the summary object contains the data.